components/Logo.vue

```diff
@@ -0,0 +1,33 @@
+<template>
```

Comment:
Is dit een default dingetje ? dan hoeft die in principe niet in master te komen 

of wil je het gebruiken als voorbeeld en aanpassen naar een ander logo ofzo?

Reply:
Is gewoon de standaard project structure van nuxt. We verwijderen alles als we aan de gang gaan. Ik maak de folders/files wel klaar voor donderdag/zaterdag om alles uit te leggen

Reply:
Okee. zo'n vermoedde had ik al. In principe is het wel de bedoeling om dit soort dingen uit master te houden. maakt voor nu even niet uit, omdat het qua structuur wel nice is om te hebben.

Reply:
Oke ff om verwarring te voorkomen. Nuxt JS werkt alleen maar met deze folders. Zonder de folders werkt het framework niet (deze folders bieden voordelen als routing, makkelijk components importen, style elements makkelijk aanpassen enz.). Dus ik verwijder alleen de standaard files maar niet de folders